package-lock.json

```diff
@@ -0,0 +1,2432 @@
+{
+  "name": "basemaps-config",
+  "version": "1.0.1",
+  "lockfileVersion": 2,
```

Comment:
what version of npm/node are you using ? `v3` is the current lockFile version

Reply:
![image](https://github.com/linz/basemaps-config/assets/12163920/2f6fa2f8-5ce2-4d06-997a-90f3df7bf7e3)
On v18.7

Reply:
Got my node updated and rerun the install. On v3 now.